addons/l10n_ar/wizards/account_move_change_rate.py

```diff
@@ -0,0 +1,33 @@
+from odoo import fields, models, api, _
+
+
+class AccountMoveChangeRate(models.TransientModel):
+    _name = 'account.move.change.rate'
+    _description = 'account.move.change.rate'
+
+    @api.model
+    def get_move(self):
+        move = self.env['account.move'].browse(
+            self._context.get('active_id', False))
+        return move
+
+    currency_rate = fields.Float(
+        'Currency Rate',
+        required=True,
+        digits=(16, 6),
+        help="Select a rate to apply on the invoice"
+    )
+    move_id = fields.Many2one(
+        'account.move',
+        default=get_move
+    )
+
+    @api.onchange('move_id')
+    def _onchange_move(self):
+        self.currency_rate = self.move_id.l10n_ar_currency_rate or self.move_id.computed_currency_rate
```

Comment:
(l10n_ar_)computed_currency_rate vs l10n_ar_currency_rate: can we make it more confusing?

Reply:
Agreed, it's confusing to have two fields.
We need one to show to the user which would be the rate used when posting (the actual rate). Computed
We need a second one to show the real rate used or the forced one.

I didn't found any other simpler way. It's just for view purpose. We can discuss next Thursday to find a better approach